client/src/components/ProfileSettings.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+
+const ProfileSettings = () => {
```

Comment:
Organizational suggestion: I would move all of these components into `components/Profile` and remove Profile from all of these component names.